app/grandchallenge/notifications/filters.py

```diff
@@ -48,3 +50,73 @@ def search_filter(self, queryset, name, value):
                 or_, [Q(**{f"{f}__in": name_qs}) for f in search_fields], Q(),
             )
         )
+
+
+FOLLOW_CHOICES = (
+    ("110", "Forum subscriptions"),
+    ("121", "Topic subscriptions"),

```

Comment:
I think that you have hardcoded the content type primary key here, right? I don't think that you can do that as those IDs are unlikely to be the same in production. Another approach is needed.